src/qt/rpcconsole.cpp

```diff
@@ -274,6 +277,19 @@ RPCConsole::RPCConsole(const PlatformStyle *platformStyle, QWidget *parent) :
     connect(ui->fontSmallerButton, SIGNAL(clicked()), this, SLOT(fontSmaller()));
     connect(ui->btnClearTrafficGraph, SIGNAL(clicked()), ui->trafficGraph, SLOT(clear()));
 
+    //Setup autocomplete and attach it
+    QStringList wordList;
+    for (int i =0; i < 53; ++i)
+    {
+        wordList<<vRPCCommands[i].name.c_str();
+    }
+    for (int i =0; i < 43; ++i)
+    {
+        wordList<<vWalletRPCCommands[i].name.c_str();
```

Comment:
You'd have to guard this. Try https://github.com/bitcoin/bitcoin/blob/317462123f8e41fd7dd967ab907e59ddffb19898/src/init.cpp#L951